test/support/utility/ffmpeg_video_generator.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Membrane.VideoCompositor.Test.Support.Utility do
+defmodule Membrane.VideoCompositor.Test.Support.Utility.FFmpegVideoGenerator do
```

Comment:
I don't like calling this `FFmpegVideoGenerator`, since it has general utility functions, such as `compare_with_error`

Reply:
yup!